test/screens/v2/widget_instance/reconstructed_alert_test.exs

```diff
@@ -228,13 +330,20 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlertTest do
     setup @valid_alert_setup_group
 
     test "returns takeover for a closure alert at this station", %{widget: widget} do
+      widget = put_is_full_screen(widget, true)
       assert [1] == WidgetInstance.priority(widget)
       assert [:full_body] == WidgetInstance.slot_names(widget)
       assert :reconstructed_takeover == WidgetInstance.widget_type(widget)
     end
```

Comment:
Ahhh hm, this test made me realize that naming is getting a little bit hairy. This test is about takeover alerts, and `is_full_screen` is set to true. When `is_full_screen` is true, that may mean it's a takeover or "the other kind", so then we run `takeover_alert` in the widgetInstance file to decide whether it's a takeover or "the other kind". One problem is that "the other kind" is sorta called "fullscreen" alert, because all of those serializer functions are called `serialize_fullscreen_alert`, which will never be run for a takeover, even though it's a "fullscreen" alert, too. The same words are getting repeated for different situations that I think will make this code hard to use in the future.

Couple related thoughts: is `takeover_alert?` still a good function to be in the WidgetInstance, or should it be in the candidateGenerator?

I like the name of `serialize_inside_flex_alert` because it's so clear what that function is for. Should the other flex serializers (outside and boundary) also be renamed to have "flex" in it?